indra/newview/app_settings/grids.xml

```diff
@@ -953,7 +904,7 @@
                 <string>agent</string>
             </array>
         <key>loginpage</key>
-            <string>https://phoenixviewer.com/app/loginV3/</string>
+            <string>http://viewer-login.agni.lindenlab.com</string>
```

Comment:
???

Reply:
The explanation for this is in the blog Austin linked to this 

> Refinement of handling of Second Life Beta Grid Entry/Splash Page (Aditi)
> 
> Firestorm uses a custom “loginpage” entry/splash page for the main (Agni) grid at https://phoenixviewer.com/app/loginV3/ rather than the usual Linden Lab provided entry at http://viewer-login.agni.lindenlab.com/
> 
> But it also uses this same URL for the Second Life Beta grid (Aditi) which I think is not as helpful as the redirect to a Beta grid specific entry page that is used from http://viewer-login.agni.lindenlab.com when the Second Life Beta grid is selected gives more useful user information when the Second Life Beta grid is selected.

Seems reasonable for the small number that might use it, I am not convinced that we should be removing our own splash though.

Reply:
Is there a dedicated Aditi login page? Because even in that address above, I see "agni". Our splash uses the same endpoints as the Lab's - this - one. I see no gain here.

Reply:
I believe that the agni page presents a different page when the grid is set to aditi, I've not tested that though.  @aiaustin has I presume.

Reply:
Yes. I also questioned why LL used an agni URL for the aditi splash page. I assumed it was simple redirect and wanted to put that in grids.xml to make the operation and intention clearer.  But I checked and received good help on the SL Discord viewer channel. The agni page has logic to serve the beta page.

The custom Firestorm page for agni works great, but it really is not helpful when selecting the Beta grid. I considered no including that change or just suggestion it in a  comment so the FS devs could decide on what was best. But as I have modified mine own setup for years to make that work better for Beta I included with clear rationale.  Since I was tidying up grids.xml and we may not get another chance I thought I would include that suggestion.

Its easy to change that line back if anyone things the FS splash page works well for the Beta grid user logins.

Reply:
For documentation this is part of the advice from the Discord Second Life Server "#viewers" channel...

You will find on that the agni page has a whole host of elements of class or id "non-prod-grid-msg" like this one:

```
<div id="non-prod-grid-msg" style="display: none">
  <img class="non-prod-grid-msg-img" src="/assets/uc-hell-yes-b4b2b4c99c72a253d94b2a26adec2642b75ab07a67fe4c9ce3a6331b321750bc.gif"/>
  <h1 id="non-prod-grid-msg-title">Test Grid: </h1>
  <img class="non-prod-grid-msg-img" src="/assets/uc-hell-yes-b4b2b4c99c72a253d94b2a26adec2642b75ab07a67fe4c9ce3a6331b321750bc.gif"/>
  <div id="non-prod-grid-msg-desc"></div>
</div>
```

Reply:
Of course an option is to provide two separate Firestorm related splash pages (simplest) or use similar web page logic and present a Beta version of the web page (rather obscure).

Reply:
> I believe that the agni page presents a different page when the grid is set to aditi, I've not tested that though. @aiaustin has I presume.

![FS-with-Beta-Grid-Selected](https://github.com/FirestormViewer/phoenix-firestorm/assets/4630885/eae178bc-5c03-44d1-99d9-22a838684bb8)

Reply:
I believe that this will suffice: https://phoenixviewer.com/app/loginV3/ vs https://phoenixviewer.com/app/loginV3/?grid=Aditi. Since the viewer already sends the 'grid' in the URL parameter, you can access the updated Aditi splash page even on the current official release of Firestorm. The information text is hardcoded; I assume that the message will not change too often (if at all).

![obraz](https://github.com/FirestormViewer/phoenix-firestorm/assets/348451/4208886d-75c0-419f-8edb-2132cfef6c36)

Reply:
I agree, that is a great improvement.  I will revert that now.